frontend/integration-tests/tests/deploy-image.scenario.ts

```diff
@@ -33,19 +33,6 @@ describe('Deploy Image', () => {
       expect(element(by.css(dropdown)).getText()).toEqual('Application: all applications');
     });
 
-    it('can be used to search for an image', async () => {
```

Comment:
We had an integration test for Deploy Image?

cc @christianvogt

Reply:
deploy-image flow existed before dev-console and then it was updated and moved.
old e2e tests were updated.
We should eventually move these to the dev-console integration tests folder and update them further.

Reply:
Instead of removing the test, can we update it to work with the new ui components.

Reply:
@christianvogt This test is obsolete now because It was testing the content in SearchResult and SearchStatus component and both of these components are removed now. Even With the new component, there is a test that covers the whole deploy image flow.

Reply:
fair enough that there's one testing the full deploy.
I just thought that instead of testing the results block, it could similarly test for the validation message.

Reply:
added test for new search field